### PR TITLE
Update Figures with Feature-Selected and JUMP-Aligned Models 

### PR DESCRIPTION
**Pull request by axiomcura**
This PR reruns all modules (0-4) and updates the figures. Specifically:

- Confusion matrices and F1 scores are now generated using the feature-selected cell-injury models.
- Probability ridge plots are generated exclusively from models trained on JUMP-aligned, feature-selected cell-injury profiles.

### CODE REVIEW

File: notebooks/4.visualization/nbconverted/4.visualization.r [hunk 1]
```diff
@@ -22,19 +22,19 @@ load_image <- function(path) {
 
 # getting all paths
 # confusing matrix paths
-confusion_matrix_path <- file.path("../../results/2.modeling/confusion_matrix.csv.gz")
+confusion_matrix_path <- file.path("../../results/2.modeling/fs_confusion_matrix.csv.gz")
```
Comment (gwaybio) [inline notebooks/4.visualization/nbconverted/4.visualization.r:25]: please make sure all of the original files (without the `fs` prefix) are no longer in the repo

File: notebooks/4.visualization/nbconverted/4.visualization.r [hunk 2]
```diff
@@ -22,19 +22,19 @@ load_image <- function(path) {
...
-pr_file_path <- file.path("../../results/2.modeling/precision_recall_scores.csv.gz")
+pr_file_path <- file.path("../../results/2.modeling/fs_precision_recall_scores.csv.gz")
 
 # probability plot
-cyto_proba_path <- file.path("../../results/3.jump_analysis/cytoskeletal_proba_scores.csv.gz")
+cyto_proba_path <- file.path("../../results/3.jump_analysis/JUMP_cytoskeletal_proba_scores.csv.gz")
 
 # injury probabilities
-injury_proba_path <- file.path("../../results/3.jump_analysis/all_injury_proba.csv.gz")
+injury_proba_path <- file.path("../../results/3.jump_analysis/JUMP_all_injury_proba.csv.gz")
```
Comment (gwaybio) [inline notebooks/4.visualization/nbconverted/4.visualization.r:37]: love the updated prefix here! super clear

File: notebooks/4.visualization/nbconverted/4.visualization.r [hunk 3]
```diff
@@ -242,20 +242,20 @@ final_model_cm <- cm_df %>%
 
 final_model_cm$dataset_type <- factor(final_model_cm$dataset_type, levels = c("Train", "Test", "Plate holdout", "Well holdout"))
 
-# Define the desired order of x-axis labels
+# # Define the desired order of x-axis labels
```
Comment (gwaybio) [inline notebooks/4.visualization/nbconverted/4.visualization.r]: looks like these comments are unnecessary?

**Review by gwaybio — APPROVED** [review]
LGTM, well done @axiomcura ! 👏

File: notebooks/4.visualization/nbconverted/4.visualization.r (hunk 3 above)
Comment (gwaybio) [reply notebooks/4.visualization/nbconverted/4.visualization.r]: (I mean the extra `#`)

Comment (axiomcura) [reply notebooks/4.visualization/nbconverted/4.visualization.r]: removed! thanks!

File: notebooks/4.visualization/nbconverted/4.visualization.r (hunk 1 above)
Comment (axiomcura) [reply notebooks/4.visualization/nbconverted/4.visualization.r:25]: Just looked at all the results folder and all files have the `fs_` and the `aligned` prefix

**Comment by axiomcura** [comment]
Woo~ merging!

**Pull request merged**